gapis/memory/pool.go

```diff
@@ -37,8 +37,8 @@ import (
 // data occur.
 type Pool struct {
 	writes  poolWriteList
-	OnRead  func(Range, uint64)
-	OnWrite func(Range, uint64)
+	OnRead  func(Range, uint64, uint64)
```

Comment:
Would be lovely to have some documentation on these parameters - even a name would go a long way.